### PR TITLE
Question read funct

### PR DESCRIPTION
**Pull request by BlueMemfis**

### CODE REVIEW

File: app/javascript/components/App.js
Comment (zan-rosenthal) [inline app/javascript/components/App.js:58]: The formatting is a bit off here. If you setup Prettier in VSCode, it should auto-format it on save which will fix this. Here's the docs for the package: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode. Feel free to merge this if it's holding up other work, but make sure to do that. Having something autoformat code for you will lead to more consistent code.

File: spec/requests/questions_spec.rb
Comment (zan-rosenthal) [inline spec/requests/questions_spec.rb:11]: There aren't any tests here. You should maybe add a test, but if you aren't going to, you can just remove this file.

File: app/javascript/pages/QuestionIndex.js
Comment (zan-rosenthal) [inline app/javascript/pages/QuestionIndex.js:5]: Please remove this console.log unless you still need it.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]

File: app/javascript/pages/QuestionIndex.js
Comment (ghost) [reply app/javascript/pages/QuestionIndex.js:5]: HI Zan, sorry but yes I would really like to keep the console.logs while we're still developing if that's okay?

File: spec/requests/questions_spec.rb
Comment (ghost) [reply spec/requests/questions_spec.rb:11]: Hi Zan, we started on requesting testing but decided to put it on hold for now. We would like to just keep it since eventually it will be coded out.

File: app/javascript/components/App.js
Comment (ghost) [reply app/javascript/components/App.js:58]: Thanks Zan, we're running into dependency issues so we are trying to just push this working code. I will work with Shaun on adding the auto format on save.

**Review by zan-rosenthal — APPROVED** [review]

**Pull request merged**
